docs/reference/adobe/experience/customerJourneyManagement/message-interaction.schema.json

```diff
@@ -25,11 +25,13 @@
                     "enum": [
                         "click",
                         "open",
-                        "action"
+                        "action",
+                        "spam_complaint"
                     ],
                     "meta:enum": {
                         "click": "Link in a message is clicked by the user.",
-                        "open": "Either Message is open by the user, or in case of push, clicking the message opened the mobile application."
+                        "open": "Either Message is open by the user, or in case of push, clicking the message opened the mobile application.",
```

Comment:
Do you also want to add a meta:enum for "action" ?

Reply:
Hi @mjasrotia,
I would not like to add it as I am not sure what scope does the author of the change wanted to have for “action”.